v2/creating-a-promotion.rst

```diff
@@ -1,4 +1,56 @@
 Create a promotion
 ==================
 
-@todo
+Imagine that your store is currently running a special New Year campaign where you'd like to give customers a 10% discount on orders worth $50 or more. With Commerce Core, it is quite simple.
+
+Adding a New Promotion
+----------------------
+
+- We can create a new promotion by going to ``/admin/commerce/promotions`` and clicking the "Add a new promotion" button.
+
+.. image:: https://user-images.githubusercontent.com/11439056/28079277-4c4d0d2a-6685-11e7-9888-1c5eeeaf7a3a.png
```

Comment:
@mglaman @joshmiller83 Thanks for your reviews guys. Really appreciate it. I'll go ahead and make the changes and get the images in as well.